docs/book/src/reference/completion.md

```diff
@@ -0,0 +1,29 @@
+# Enable shell auto completion
+
```

Comment:
The Kubebuilder completion script for Bash can be generated with the command `kubebuilder completion bash` as the Kubebuilder completion script for Zsh can be generated with the command `kubebuilder completion zsh`. 

Note that sourcing the completion script in your shell enables Kubebuilder autocompletion. 

However, the completion Bash script depends on bash-completion, which means that you have to install this software first (you can test if you have bash-completion already installed by running type _init_completion).